include/PreferenceList.h

```diff
@@ -9,6 +9,8 @@
 #include <vector>
 #include "TDefs.h"
 
+enum PreferenceOrderT { cEqual=0, cBetter, cWorse };
```

Comment:
Indentation should be:
enum PreferenceOrderT {
  cEqual=0,
  cBetter,
  cWorse
};